.github/workflows/push-prod-environment.yaml

```diff
@@ -75,6 +75,11 @@ jobs:
           HAPI_RATE_HISTORY: ${{secrets.HAPI_RATE_HISTORY }}
           HAPI_HYPERION_START_AT: '2022-07-01T00:00:00.000+00:00'
           HAPI_EDEN_ELECTION_INTERVAL: 7889400
+          HAPI_EDEN_HISTORIC_ELECTION: "[
```

Comment:
Load this variable from secrets.

It is not a secret value but it is easier to update only a variable than having to update the code en redeploying